doc/installation_instructions/platform/olcf-summit.rst

```diff
@@ -50,7 +50,7 @@ directory from which these instructions are executed.
   # install PyTorch and TensorFlow backend for the Orchestrator database.
   export Torch_DIR=/ccs/home/$USER/.conda/envs/$ENV_NAME/lib/python3.9/site-packages/torch/share/cmake/Torch/
   export CFLAGS="$CFLAGS -I/ccs/home/$USER/.conda/envs/$ENV_NAME/lib/python3.9/site-packages/tensorflow/include"
-  export SMARTSIM_REDISAI=1.2.5
+  export SMARTSIM_REDISAI=1.2.7
```

Comment:
Do we need to update the site install on summit if we are going to make this change to the docs? Should be put in a ticket to do that sometime before we release?

Reply:
I updated this script and I think it will work, even though there is no matching IBM version of PyTorch available (but I remember 1.10 being largely compatible). We will open a ticket for the site install and make sure it works.